tests/unit/sceneUpdateTests.cpp

```diff
@@ -42,210 +42,192 @@ bool loadConfig(const std::string& _sceneString, Node& root) {
-    CHECK(root["seq"][1].Scalar() == "global_a_value");
-    CHECK(root["map"]["b"].Scalar() == "newer_global_b_value");
+
+    // // Apply updated globals.
+    // SceneLoader::applyGlobals(config);
+    // CHECK(root["seq"][1].Scalar() == "global_a_value");
+    // CHECK(root["map"]["b"].Scalar() == "new_global_b_value");
+    // // Add an update.
+    // updates = {{"global.b", "newer_global_b_value"}};
+    // // Apply the update.
+    // SceneLoader::applyUpdates(config, updates);
+    // CHECK(root["global"]["b"].Scalar() == "newer_global_b_value");
+    // // Apply updated globals.
+    // SceneLoader::applyGlobals(config);
+    // CHECK(root["seq"][1].Scalar() == "global_a_value");
+    // CHECK(root["map"]["b"].Scalar() == "newer_global_b_value");
 }
 
 TEST_CASE("Regression: scene update requesting a sequence from a scalar") {
-
-    // Setup.
-    MockPlatform platform;
-    Scene scene(platform, Url());
-    REQUIRE(loadConfig(sceneString, scene.config()));
+    Node config;
+    REQUIRE(loadConfig(sceneString, config));
     // Add an update.
     std::vector<SceneUpdate> updates = {{"map.a#0", "new_value"}};
     // Apply scene updates, reload scene.
-    SceneLoader::applyUpdates(platform, scene, updates);
-    const Node& root = scene.config();
+    SceneLoader::applyUpdates(config, updates);
+    const Node& root = config;
 
     // causes yaml exception 'operator[] call on a scalar'
 }
 
 TEST_CASE("Scene update statuses") {
-    MockPlatform platform;
-    Scene scene(platform, Url());
-    REQUIRE(loadConfig(sceneString, scene.config()));
-    Node& root = scene.config();
-
-    std::vector<SceneUpdate> updates = {{"map.a", "{ first_value"}};
-    CHECK(SceneLoader::applyUpdates(platform, scene, updates) == false);
-    CHECK(scene.errors.front().error == Error::scene_update_value_yaml_syntax_error);
-    scene.errors.clear();
-
-    updates = {{"someKey.somePath", "someValue"}};
-    CHECK(SceneLoader::applyUpdates(platform, scene, updates) == false);
-    CHECK(scene.errors.front().error == Error::scene_update_path_not_found);
-    scene.errors.clear();
-
-    updates = {{"map.a.map_a_value", "someValue"}};
-    CHECK(SceneLoader::applyUpdates(platform, scene, updates) == false);
-    CHECK(scene.errors.front().error == Error::scene_update_path_not_found);
-    scene.errors.clear();
-
-    updates = {{"!map#0", "first_value"}};
-    CHECK(SceneLoader::applyUpdates(platform, scene, updates) == false);
-    CHECK(scene.errors.front().error == Error::scene_update_path_not_found);
-    scene.errors.clear();
-
-    updates = {{"key_not_existing", "first_value"}};
-    CHECK(SceneLoader::applyUpdates(platform, scene, updates) == true);
-
-    updates = {{"!map#0", "{ first_value"}};
-    CHECK(SceneLoader::applyUpdates(platform, scene, updates) == false);
-    CHECK(scene.errors.front().error == Error::scene_update_value_yaml_syntax_error);
-    scene.errors.clear();
+    {
+        Node config;
```

Comment:
Just FYI you can avoid repeating "setup" code by putting the different conditions to check in `SECTION`s https://github.com/catchorg/Catch2/blob/master/docs/tutorial.md#test-cases-and-sections

Reply:
Oh that's handy!